packages/bindgen/src/templates/node.ts

```diff
@@ -99,8 +99,8 @@ class NodeAddon extends CppClass {
 
             DefineAddon(exports, {
                 ${Object.entries(this.exports)
-            .map(([name, val]) => `InstanceValue(${name}, ${val}.Value(), napi_enumerable),`)
-            .join("\n")}
+                  .map(([name, val]) => `InstanceValue(${name}, ${val}.Value(), napi_enumerable),`)
```

Comment:
I have no idea what is going on, since `eslint --fix` doesn't change this file for me either before or after your change. Is there any chance you have an additional formater running?

I was concerned that it would end up flip-flopping, but since it doesn't, leave this in.